### DISCUSSION TITLE
Bed leveling

### DISCUSSION THREAD

**Discussion by Wolfy64** [NARUTOfzr/Neptune_3#21] [Q&A] [answered]
It would be great to know how to fix the bed leveling and print a proper test. 🙇‍♂️

**Comment by NARUTOfzr** [comment, marked as answer]
#10 
Try it first.

**Reply by Wolfy64** [reply]
I already tried #10. To be honest, I don't get how, loosening wheels will fix it, I did not notice any change. Maybe I did not do it the right way.
Will try again once my print is finished.
PS: I tried all FW version up to V1_1.0.4
PS2: I did all the process again and still can't make it, always the top right corner that is a bit off
![IMG_1219 Large](https://user-images.githubusercontent.com/25457563/194219572-88f9fe0f-2931-4211-be66-0a2eee332239.jpeg)

**Reply by NARUTOfzr** [reply]
![image](https://user-images.githubusercontent.com/64059773/194319973-f87cfd26-c50f-459b-8895-8167682789c8.png)
Are the three V wheels here tightened?
The three V-wheels on the other side, they turn easily when you print, right?

If the above conditions are normal, then please take a picture of the leveling value to see.

**Reply by Wolfy64** [reply]
All the conditions above are met, I did it one more time #10 and this time I also did #20 and everything still the same.
Here my latest leveling value and also my latest test:
![IMG_1232 Medium](https://user-images.githubusercontent.com/25457563/194671769-fd0783d1-ad7f-4d6f-96ef-07fa49f3510d.jpeg)
![IMG_1233 Medium](https://user-images.githubusercontent.com/25457563/194671958-3fbd507f-80b0-482d-adc2-960764984fa7.jpeg)

**Reply by NARUTOfzr** [reply]
#7 I hope I can use their luck to help you solve this problem.
![image](https://user-images.githubusercontent.com/64059773/194716056-71e27e9d-dc1d-4bcc-8d1d-f4dc53a6934b.png)
![image](https://user-images.githubusercontent.com/64059773/194716090-bd8881ff-feaf-4d4b-b429-1f9b146813c7.png)

- &
I hope you don't mind my repeating the story. Try to loosen or remove the right V wheel as shown below. (Tighten the other side properly)
![image](https://user-images.githubusercontent.com/64059773/194716978-c7031283-875b-49d7-870a-b458897adfd1.png)

**Reply by Wolfy64** [reply]
It works! 🙇
I loosened the right V wheel a bit more, when the doc said loosen I did not know that mean almost not tight at all.
Here is the result that is not perfect, but I think it should be good enough 🙂
![IMG_1235 Medium](https://user-images.githubusercontent.com/25457563/194780146-404d6337-41ed-4efa-9d88-c64f9af28a9d.jpeg)
![IMG_1236 Medium](https://user-images.githubusercontent.com/25457563/194780147-1eb2b931-39c3-4b95-924c-6a8e2049b90e.jpeg)
